DEHPCommon/UserPreferenceHandler/UserPreference.cs

```diff
@@ -0,0 +1,35 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="UserPreference.cs"company="RHEA System S.A.">
+//    Copyright(c) 2020 RHEA System S.A.
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Kamil Wojnowski, Nathanael Smiechowski.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace DEHPCommon.UserPreferenceHandler
+{
+    /// <summary>
+    /// Base class from which all <see cref="UserPreference"/> shall derive
+    /// </summary>
+    public class UserPreference
+    {
+        /// <summary>
+        /// Gets or sets server serverType value
+        /// </summary>
+        public string ServerType { get; set; }
```

Comment:
All these properties should be in own class that is storesd in a list here. You will save lots of servers in the user preferences. Also, no Password.